### PR TITLE
add a blog post: Introducing The Java Batch Processing API And JBeret Implementation

### PR DESCRIPTION
**Pull request by liweinan**

### CODE REVIEW

**Comment by liweinan** [comment]
@chengfang Could you please help to check this blog post? Thanks!

File: _posts/2022-11-11-jberet-intro.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout:     post
+title:      "Introducing The Java Batch Processing API And JBeret Implementation"
+subtitle:   ""
+date:       2022-11-01
```
Comment (chengfang) [inline _posts/2022-11-11-jberet-intro.md]: is this an old post back in 2022?  Why is it empty?

Comment (liweinan) [reply _posts/2022-11-11-jberet-intro.md]: oops, I'll fix it :D

**Comment by liweinan** [comment]
@chengfang I have removed the useless file. If the blog post is okay I'll squash the commits and publish it.

**Review by chengfang — APPROVED** [review]

**Pull request merged**